gen/definitions/ntp_server.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: NTP Server
+class_name: datetimeNtpProvider
+dn: sys/time/prov-[%s]
+ds_description: This data source can read an ntp server or peer.
+res_description: This resource can manage an ntp server or peer.
+doc_path: System/top:NTP/
```

Comment:
`doc_path` is the relative path to the reference documentation of the class. If you go here (https://pubhub.devnetcloud.com/media/dme-docs-10-2-2/docs) and search for the class (`datetime:NtpProvider `), you should find the class documentation and should be able to extract the relative path from the URL. In this case it would be `System/datetime:NtpProvider/`.